src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_AzDoFlow.cs

```diff
@@ -1,11 +1,10 @@
-using NUnit.Framework;
 using System;
 using System.Threading.Tasks;
+using NUnit.Framework;
 
 namespace Maestro.ScenarioTests
 {
     [TestFixture]
-    [NonParallelizable]
```

Comment:
Did we mean to remove this?  We really need it at the assembly level. All of these tests implicitly depend on Environment.CurrentDirectory, I think, so it'll get really weird if any of the tests run together.

Reply:
Nope, I didn't mean to, that happened in the merge.